Makefile

```diff
@@ -196,7 +196,7 @@ docker-build-scheduler: ## Build docker image for (autoscaling) scheduler
 
 .PHONY: docker-build-examples
 docker-build-examples: vm-informant bin/vm-builder ## Build docker images for testing VMs
-	./bin/vm-builder -src postgres:15-bullseye -dst $(E2E_TESTS_VM_IMG)
+	./bin/vm-builder -src postgres:15-bullseye -dst $(E2E_TESTS_VM_IMG) -enable-informant
```

Comment:
Need to also `-enable-monitor`?

Reply:
Yup, just realized this. Am testing locally as it should be a little faster.